statsmodels/tsa/stattools.py

```diff
@@ -1681,6 +1680,9 @@ def kpss(x, regression='c', lags=None, store=False):
     else:
         lags = int(lags)
 
+    if lags > nobs:
```

Comment:
What about the number of obs and the included deterministic terms? This is unlikely to bind but could create an error.

Reply:
I'm not sure I follow, could you please explain this a little more? My understanding is that the error that occurs when lags exceeds # obs stems from `_sigma_est_kpss`, particularly:

`resids_prod = np.dot(resids[i:], resids[:nobs - i])`

Example error with # obs = 3, lags set to 4:

> ValueError: shapes (0,) and (2,) not aligned: 0 (dim 0) != 2 (dim 0)